src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -36,6 +36,10 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
         return this.databaseName;
     }
 
+    public get description(): string {
+        return ext.connectedPostgresDB && ext.connectedPostgresDB.fullId === this.fullId ? 'Connected' : '';
```

Comment:
Does the node get refreshed when people connect/disconnect a database?

Reply:
Yeah both the old node and the new connected node get refreshed